### PR TITLE
Initial Dart Web App launch configuration, currently commented out

### PR DESCRIPTION
**Pull request by jwren**

### CODE REVIEW

File: Dart/src/com/jetbrains/lang/dart/ide/runner/web/DartWebRunConfigurationType.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  }
+
+  protected DartWebRunConfigurationType() {
+    super("DartWebRunConfigurationType", DartBundle.message("runner.web.configuration.name"),
+          DartBundle.message("runner.web.configuration.description"),
+          DartIcons.Dart_web);
+    addFactory(new ConfigurationFactory(this) {
+      @Override
+      public String getName() {
+        return "Dart Web App"; // compatibility
```
Comment (alexander-doroshko) [inline Dart/src/com/jetbrains/lang/dart/ide/runner/web/DartWebRunConfigurationType.java]: Compatibility? ;).

**Comment by alexander-doroshko** [comment]
> currently commented out

doesn't look like this :)

Anyway, it is too early to include it in the Dart plugin. Need to understand what we want first.

**Comment by alexander-doroshko** [comment]
Looks like we can close this.

**Comment by jwren** [comment]
Yes, thanks.

**Pull request closed without merging**